### ISSUE TITLE
[2.13.5] Imports show results that are already in scope

### ISSUE THREAD

**Issue by tgodzik** [scalameta/metals#2546]
**Describe the bug**

After the update to Scala 2.13.5, import statements will offer completions that are not needed, because they are included in the scope. This might be a result of a bugfix in the compiler and this might have never actually be checked in Metals.

It does only influence 2.13.5, so instead of blocking on this I decided to go ahead with the release to unblock both Scala 2.13.5 and 3.0.0-RC1 support.
 
**To Reproduce**

```scala
package a
object Main {
  import a.Outer.Inner
  import Inner@@
}
object Outer {
  class Inner
}
```

Try to complete at `@@` position on Scala 2.13.5

**Expected behavior**

No completions are offered.

**Installation:**

- Operating system: macOS/Windows/Linux
- Editor: Visual Studio Code/Atom/Vim/Sublime/Emacs
- Metals version: v0.9.10

**Additional context**

The issue started with https://github.com/scalameta/metals/pull/2544

**Search terms**

completions duplicate

**Comment by kasiaMarek** [comment]
```scala
package a

object Main:
  import a.Outer.Inner
  import Inner.A

object Outer {
  object Inner {
    object A
  }
}
```
In this case offering a symbol already in scope might make sense. So this issue would refer only to types (`classes` and `types`).

**Comment by jozanek** [comment]
@tgodzik you can close this one as won't fix.
